client/patients/get-funded.js

```diff
@@ -33,7 +16,16 @@ angular.module('eir.getFunded', [])
 
     // only submit if form is valid
     if(form.$valid) {
-      fileUpload.uploadFileToUrl($scope.patient);
+      fileUpload.uploadFileToUrl($scope.patient)
+        .then(function (res, err){
```

Comment:
@benjaminhoffman here I am catching the promise response (if there is no error) and redirecting to their new page